internal/framework/schemas.go

```diff
@@ -9,13 +9,14 @@ import (
 	"fmt"
 	"reflect"
 
+	"github.com/nuodb/terraform-provider-nuodbaas/openapi"
```

Comment:
It is strange that the linter doesn't complain as the local imports (i.e. with prefix `github.com/nuodb/terraform-provider-nuodbaas`) should be after the 3-rd party imports.